bootkube-up

```diff
@@ -200,7 +179,7 @@ echo_green "\nComplete!"
 
 ### RENDER ASSETS:
 echo_green "\nPhase VI: Running Bootkube to render the Kubernetes assets:"
-bootkube render --asset-dir=$BOOTKUBE_DIR/.bootkube --experimental-self-hosted-etcd --etcd-servers=http://10.3.0.15:12379 --api-servers=https://$KUBE_DNS_API:$KUBE_SVC_PORT --pod-cidr=$KUBE_POD_CIDR --service-cidr=$KUBE_SVC_CIDR
```

Comment:
admittedly, i'm not sure.  i'll do some digging and see what i can find

Reply:
thanks for looking into it. i'm ok with this being in a separate PR as long as we can keep tracking it somewhere. from a work POV, it can be part of the multi-node or `bootkube-add` task.